benchmark/scenarios/read_request_load.py

```diff
@@ -2,105 +2,41 @@
 """
```

Comment:
Common code taken out. The read scenario is now quite simple, as most of the code was common and it doesn't need a setup